### PR TITLE
♻️✨ Grafana Monitoring Stack Refactoring

### PR DESCRIPTION
**Pull request by mrnicegyu11**
*This is 3/3 from split huge PR https://github.com/ITISFoundation/osparc-ops-environments/pull/117*

This PR brings:
- ✨ Adds new grafana dashboards (including s4l-lite monitoring) and updates existing ones
- ✨ Adds deployment name (FQDN) to grafana alerts
- ✨ Enables slack (`== mattermost`) webhook notifications for grafana alerts (instead of emails)
- 🐛 Enables handling of more than one grafana alert in provisioning
- 🔨 Removes postgres-datasource in grafana
- 🔨 Removes postgres grafanareader user

👷👷👷 `osparc-ops-deployment-configuration` changes 👷👷👷:
- 🔨 REMOVE - From now on unused: `POSTGRES_GRAFANA_PASSWORD`
- 🔨 REMOVE - From now on unused: `POSTGRES_GRAFANA_USER`
- 🔨 REMOVE - From now on unused: `POSTGRES_EXPORTER_DATA_SOURCE_NAME`
- ✨ ADD `GRAFANA_ALERTS_SLACK` : Webhook URL for slack/mattermost grafana alerts. Example: `GRAFANA_ALERTS_SLACK=https://mattermost.speak.com/hooks/asdfextreme` 

🚧🚧🚧 Necessary when rolling out 🚧🚧🚧:
- [ ] Remove grafana user from postgres (https://git.speag.com/oSparc/osparc-ops-environments/-/issues/512 ) 
- [ ] Remove postgres grafana user env-vars from repo.config
- [ ] Switch to slack notifications for grafana
- [ ] Re-provision all alerts and dashboards everywhere

### CODE REVIEW

**Comment by mrnicegyu11** [comment]
FYI: This was successfully tested on aws-prod

**Review by pcrespov — COMMENTED** [review]
:+1:

File: services/monitoring/grafana/provisioning/osparc.io/dashboards/simcore/s4l-lite admin overview.json
Comment (GitHK) [inline services/monitoring/grafana/provisioning/osparc.io/dashboards/simcore/s4l-lite admin overview.json:1]: maybe space ` ` in the file name can be avoided

File: services/monitoring/grafana/scripts/import.py [hunk 1]
```diff
@@ -139,14 +139,24 @@ def subsituteDatasources(
 
 def main(foldername: str = "", overwrite: bool = True):
     # Get mail adress for alerts:
-    grafanaAlertingMailTarget = env.str("GRAFANA_ALERTS_MAIL")
+    try:
+        grafanaAlertingMailTarget = env.str("GRAFANA_ALERTS_MAIL")
+    except EnvError:
+        grafanaAlertingMailTarget = None
```
Comment (GitHK) [inline services/monitoring/grafana/scripts/import.py:145]: you could use a default here: see https://github.com/sloria/environs/blob/master/environs/__init__.py#L71

```suggestion
    grafanaAlertingMailTarget = env.str("GRAFANA_ALERTS_MAIL", default=None)
```

File: services/monitoring/grafana/scripts/import.py [hunk 2]
```diff
@@ -139,14 +139,24 @@ def subsituteDatasources(
...
+    try:
+        grafanaAlertingSlackTarget = env.str("GRAFANA_ALERTS_SLACK")
+    except EnvError:
+        grafanaAlertingSlackTarget = None
```
Comment (GitHK) [inline services/monitoring/grafana/scripts/import.py:149]: Same here

**Review by GitHK — COMMENTED** [review]
Some comments

**Review by sanderegg — APPROVED** [review]

File: services/monitoring/grafana/scripts/import.py (hunk 1 above)
Comment (mrnicegyu11) [reply services/monitoring/grafana/scripts/import.py:145]: will add thx

File: services/monitoring/grafana/scripts/import.py (hunk 2 above)
Comment (mrnicegyu11) [reply services/monitoring/grafana/scripts/import.py:149]: will add thx

File: services/monitoring/grafana/provisioning/osparc.io/dashboards/simcore/s4l-lite admin overview.json
Comment (mrnicegyu11) [reply services/monitoring/grafana/provisioning/osparc.io/dashboards/simcore/s4l-lite admin overview.json:1]: will change thx

**Pull request merged**